src/Select/__tests__/Select-test.js

```diff
@@ -0,0 +1,108 @@
+jest.dontMock("../../Mixin/BindMixin");
+jest.dontMock("../../Util/Util");
+jest.dontMock("../../Util/DOMUtil");
+jest.dontMock("../Select");
+jest.dontMock("../SelectOption");
+
+/* eslint-disable no-unused-vars */
+var React = require("react");
+/* eslint-enable no-unused-vars */
+var ReactDOM = require("react-dom");
+
+var TestUtils;
+if (React.version.match(/15.[0-5]/)) {
+  TestUtils = require("react-addons-test-utils");
+} else {
+  TestUtils = require("react-dom/test-utils");
+}
+
+var Select = require("../Select.js");
+var SelectOption = require("../SelectOption.js");
+
+describe("Select", function() {
+  beforeEach(function() {
+    this.callback = jasmine.createSpy();
+    this.select = TestUtils.renderIntoDocument(
```

Comment:
Will it be possible to use Shallow rendering here? https://reactjs.org/docs/shallow-renderer.html

Reply:
not sure why, but when implementing it, the tests never finish... have to investigate this, but I think thats out of scope for this PR.